server/src/endpoints/forward.rs

```diff
@@ -69,15 +69,11 @@ fn forward_upstream(request: &HttpRequest<ServiceState>) -> ResponseFuture<HttpR
         .set_header("X-Forwarded-For", get_forwarded_for(request))
         .set_header("Connection", "close");
 
-    let forwarded_request = if request.headers().get(header::CONTENT_TYPE).is_some() {
-        tryf!(forwarded_request_builder.streaming(request.payload()))
-    } else {
-        tryf!(forwarded_request_builder.finish())
-    };
-
-    forwarded_request
-        .send()
+    ForwardBody::new(request)
```

Comment:
Was this change intentional in this PR? I'm all up for it, probably just doesn't belong here...